notebooks/MAST/plot_analyze_with_lightkurve/plot_analyze_with_lightkurve.ipynb

```diff
@@ -0,0 +1,953 @@
+{
```

Comment:
<p>Should clarify, "there are many (nearly 5000!) candidate binary systems in the Price-Whelan et al. 2018 catalog that may have eclipses..." There are many more candidates in other catalogs too!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>